ui/common/css/vendor/chessground/_board-2d.scss

```diff
@@ -16,9 +16,9 @@
 
     @if $coord-color-white and $coord-color-black {
       coords {
-        --cg-ccw: #{$coord-color-white};
-        --cg-ccb: #{$coord-color-black};
-        --cg-cs: none;
+        ---cg-ccw: #{$coord-color-white};
```

Comment:
Feel free to just scan over the rest as it's mostly search and replace, but there are some tiny changes in dasher/src/background.ts and site/src/main.ts to look at.